src/index.ts

```diff
@@ -40,6 +45,7 @@ export class LoggerModule {
         {provide: LoggerConfig, useValue: config || {}},
```

Comment:
So I was thinking something more along this lines of this: seems to be a little cleaner.

```
   static forRoot(config?: LoggerConfig, customHttpService?: NGXLoggerHttpService): ModuleWithProviders {
     return {	 
       ngModule: LoggerModule,	       
       providers: [
         {provide: LoggerConfig, useValue: config || {}},
         NGXLogger,	         
         NGXLoggerHttpService,
        {provide: NGXLoggerHttpService, useClass: customHttpService || NGXLoggerHttpService},
         CustomNGXLoggerService
       ]	
};
```